jaeger-core/src/main/java/com/uber/jaeger/Span.java

```diff
@@ -37,6 +37,7 @@
   private final Map<String, Object> tags;
   private long durationMicroseconds; // span durationMicroseconds
   private String operationName;
+  private final SpanContext parentContext;
```

Comment:
is it possible to keep this new field out of the Span and limit the impact on the SpanBuilder only?